Incompatible graphql-language-service-utils

In npm, the current version of graphql-language-service-utils is 1.2.1 which seems to have issues when resolved transitively by the other older components.

Forcing graphql-language-service-utils to 1.2.0 resolves our build issue.

Same experience on our side...
Getting this error message
```
Module not found: Error: Can't resolve 'graphql/validation/rules/ExecutableDefinitions' in '../node_modules/graphql-language-service-utils/dist'
```
solved by forcing by reverting to the last release which worked on our side (graphql-language-service-utils-1.0.18.tgz")

--- a/packages/utils/package.json
+++ b/packages/utils/package.json
@@ -26,7 +26,7 @@
     "prepublish": "node ../../resources/prepublish.js"
   },
   "peerDependencies": {
-    "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0"
+    "graphql": "^0.12.0 || ^0.13.0"
   },
   "dependencies": {
     "graphql-config": "2.0.1",

